boards/riscv32/rv32m1_vega/rv32m1_vega_ri5cy_defconfig

```diff
@@ -12,3 +12,4 @@ CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_MULTI_LEVEL_INTERRUPTS=y
+CONFIG_FLASH=y
```

Comment:
This should not be in the default board config; it should get set in an application config.

Reply:
Done